processor.js

```diff
@@ -8,6 +10,30 @@ var stylus = require('stylus');
 var logger = require('./logger');
 var utils = require('./utils');
 
+
+
+/**
+ * Router for processing files CSS and Stylus
+ */
+function process(githubClient, repo, branch, file, config, handleIncompatibility) {
+    var processor;
+
+    // Test and report on this file if it's a stylesheet
```

Comment:
We can probably remove this comment now. The function comment says about the same thing.